draft-ietf-dnsop-svcb-httpssvc.md

```diff
@@ -367,11 +367,11 @@ Registered key names should only contain characters from the ranges
 Values are in a format specific to the SvcParamKey.
 Their definition should specify both their presentation format
 and wire encoding (e.g., domain names, binary data, or numeric values).
+The initial keys and formats are defined in {{keys}}.
 
 The presentation format for SvcFieldValue is a whitespace-separated
-list of elements representing a key-value pair, with an absent value
-or "=" indicating an empty value.  Each element is presented in the
-following form:
+list of key=value pairs.  When the value, or both the value and the "=", are
+omitted, the presentation value is the empty string.
```

Comment:
The number of commas here is confusing.  Is there a way to reword this?
"When the value is omitted, or both the value and the "=" are omittied", then the presentation value is the empty string."

Reply:
Done